app/src/main/res/layout/quest_localizedname_row.xml

```diff
@@ -23,7 +23,7 @@
         android:layout_height="wrap_content"
         android:minHeight="48dp"
         android:layout_toEndOf="@id/languageButton"
-        android:layout_toStartOf="@+id/deleteButton"
+        android:layout_toStartOf="@+id/nameSuggestionsButton"
```

Comment:
Isn't this XML used by several quests? Doesn't just renaming this button break the form for the other quests that use it?

Reply:
There's no button-renaming going on here. I'm just changing the RHS end of this layout element to be the nameSuggestionsButton instead of the deleteButton. This allows the drop down to show without occlusion at the same time as text in the field. I have tested this with other quests (like NameRoad) and with and without text in this quest, and it still works correctly. It even gives the text more room when the nameSuggestionsButton is not visible, which I wasn't really expecting, but is nice to see.

Reply:
Ok!